chore(curriculum): add transcripts for styling forms module (#58443)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lecture-best-practices-for-styling-forms/672aa791227e755da64cf240.md

Watch the video lecture and answer the questions below.

# --transcript--

What are some of the best practices for styling text inputs?

> As with all text elements you need to ensure the styles you apply to the text input are accessible, this means the font needs to be adequately sized and the color needs to have sufficient contrast with the background. The placeholder however is often forgotten, it's important to remember that this is also text and you will likely need to change the styling to ensure it's readable.

Reviewer: ```suggestion
As with all text elements you need to ensure the styles you apply to the text input are accessible. This means the font needs to be adequately sized and the color needs to have sufficient contrast with the background.

The placeholder, however, is often forgotten. It's important to remember that this is also text and you will likely need to change the styling to ensure it's readable.
```

> Anothe thing to keep in mind is that you should allow the user to modify the input, for example, if it's a text area you should not remove the ability to resize it. The input should also scale accordingly when the user zooms the page.

Reviewer 2: ```suggestion
Another thing to keep in mind is that you should allow the user to modify the input, for example, if it's a text area you should not remove the ability to resize it. The input should also scale accordingly when the user zooms the page.
```

> Another thing to keep in mind is that you should allow the user to modify the input, for example, if it's a text area you should not remove the ability to resize it. The input should also scale accordingly when the user zooms the page.

Reviewer: ```suggestion
Another thing to keep in mind is that you should allow the user to modify the input. For example, if it's a text area you should not remove the ability to resize it. The input should also scale accordingly when the user zooms the page.
```

> Input elements are also focusable. When you are editing your styles you should take care that you preserve a noticable indicator when the element has focus such as a bold border. Additionally there is the error state to consider. When the user's text does not pass the input validation there is a visual indicator that shows there is something wrong. As you give your element your personal flare you want to be sure that state is still notocable and that it does not look similar to the focus indicator. With all this in mind you can now create captivating user forms.

Reviewer: ```suggestion
Input elements are also focusable. When you are editing your styles you should take care that you preserve a noticeable indicator when the element has focus such as a bold border. Additionally there is the error state to consider. When the user's text does not pass the input validation there is a visual indicator that shows there is something wrong. As you give your element your personal flair you want to be sure that state is still noticeable and that it does not look similar to the focus indicator. With all this in mind you can now create captivating user forms.
```

> Watch the video lecture and answer the questions below.

Reviewer 3: ```suggestion
Watch the video or read the transcript and answer the questions below.
```
